packages/prisma/schema.prisma

```diff
@@ -700,6 +702,33 @@ enum PaymentOption {
   HOLD
 }
 
+enum AuditLogBookingTriggerEvents {
+  BOOKING_CREATED
+  BOOKING_PAYMENT_INITIATED
+  BOOKING_PAID
+  BOOKING_RESCHEDULED
+  BOOKING_REQUESTED
+  BOOKING_CANCELLED
+  BOOKING_REJECTED
+  BOOKING_CONFIRMED
+}
+
+enum AuditLogTriggerTargets {
+  BOOKING
+  WEBHOOKS
+  API_KEYS
+  SCHEDULE
+  EVENT_TYPES
+  TEAMS
+  USER_PROFILE
+  PASSWORD
+  IMPERSONATION
+  APPS
+  ROUTING_FORMS
+  WORKFLOWS
+  SETTINGS
+}
+
```

Comment:
Following Webhook feature design, I created these enums. They are used throughout the feature for typesafety, to render the eventToggles UI (where users can disable/enable event reporting) and for UI translations.